docs/assets/documents/governance-for-hospitalNotification.md

```diff
@@ -0,0 +1,111 @@
+
+
+### 1.2 VANS-network and Fælleskommunal infrastructure
+The following sections describe the flow of a HospitalNotification message. There are three types of receiving systems that can receive a HospitalNotification: Type A, B and C. Since there is different requirements to acknowledging of a HospitalNotification, they are presented seperately.  
+
+The flow of events and rules, when sending and receiving a HospitalNotification, are common for them all systems. However, a type C system shall not acknowledge a HospitalNotification.
+
+|Rules describing the flow of events when sending, and acknowledging a HospitalNotification message|
+|:---|
+|A Sending EcoSystem **SHALL** wrap a HospitalNotification in a VANS-envelope|
+|The Fælleskommunal infrastructure **SHALL** unwrap a HospitalNotification in a VANS-envelope|
+|The Fælleskommunal infrastructure **SHALL** wrap a HospitalNotification in a Beskedfordeler-envelope|
+|A Receiving EcoSystem **SHALL** unwrap a HospitalNotification in a Kombit beskedfordeler kuvert|
+|A Receiving EcoSystem **SHALL** wrap an Acknowledgement in a Kombit beskedfordeler kuvert|
+|The Fælleskommunal infrastructure **SHALL** unwrap a Acknowledgement in a Beskedfordeler-envelope|
+|The Fælleskommunal infrastructure **SHALL** wrap a Acknowledgement in a VANS-envelope|
+|A Sending EcoSystem **SHALL** unwrap an Acknowledgement in a VANS-envelope|
+
+
+#### Type A
+A Type A system is according to KOMBIT a receiving system that also shall receive and acknowledge a HospitalNotification. The acknowledgement of a Type A system is the basis for the Fælleskommunal infrastructure to sent an Acknowledgement to the sending system. 
+
+(INSERT sequencediagram)
+
+
+#### Type B
+A Type B system is according to KOMBIT a receiving system that also shall receive and acknowledge a HospitalNotification. The acknowledgement of a Type B system isn't the basis for the Fælleskommunal infrastructur to sent an Acknowledgement to the sending system.
+
+(INSERT sequencediagram)
+
+#### Type C
+A Type C system is according to KOMBIT a receiving system that is able to  receive a HospitalNotification, but not to acknowledge it.
+
+(INSERT sequencediagram)
+
+
+### 1.3 Acknowledgements
+
+All MedCom FHIR messages **SHALL** be acknowledged, which is also valid for a HospitalNotification message. To acknowledge a HospitalNotification message the [MedCom FHIR Acknowledgement](https://medcomdk.github.io/dk-medcom-acknowledgement/) standard **SHALL** be used.
+
+### 1.4 Envelopes
+
+When the HospitalNotification message is sent over the VANS-network, it **SHALL** be in a VANS-envelope. [This page describes the use of VANS-envelope](/docs/assets/documents/030_Governance-for-Network-Layer.md)
```

Comment:
it **SHALL** be in a VANS-envelope.
erstattes med 
it **SHALL** be wrapped in a VANS-envelope.